useful_scripts/rg_replace.sh

```diff
@@ -24,6 +24,14 @@
 #   3. `gs_rgr`
 #   4. `gs_rg_replace`
 
+# SIMPLE FISH IMPLEMENTATION (not using bins)
```

Comment:
What do you mean: "not using bins"?

Reply:
binaries, I haven't played much with binaries and so I just point to the .sh file when calling the function.